controllers/message-utils.js

```diff
@@ -51,45 +128,67 @@ module.exports = {
       callback(null, msgs);
     });
   },
-  updateMessage: function(id, body, callback) {
-    module.exports.getMessage(id, function(err, msg) {
+  /*
+   * See: updateMessageTaskStates
```

Comment:
Probably not. I was trying to be thorough because I found how `getMessages` wasn't the plural or `getMessage` really confusing.